projects/phone/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "@maskito/phone",
+    "version": "1.4.0",
+    "description": "The optional framework-agnostic Maskito's package with phone masks",
+    "keywords": [
+        "mask",
+        "inputmask",
+        "phonemask",
+        "phone",
+        "javascript",
+        "typescript"
+    ],
+    "homepage": "https://maskito.dev",
+    "bugs": "https://github.com/taiga-family/maskito/issues",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/taiga-family/maskito.git"
+    },
+    "license": "Apache-2.0",
+    "author": {
+        "name": "Vladimir Potekhin",
+        "email": "vladimir.potekh@gmail.com",
+        "url": "https://github.com/vladimirpotekhin"
+    },
+    "contributors": [
+        "Alex Inkin <alexander@inkin.ru> (https://twitter.com/waterplea)",
+        "Maksim Ivanov <splincodewd@yandex.ru>",
+        "Vladimir Potekhin <vladimir.potekh@gmail.com>",
+        "Nikita Barsukov <nikita.s.barsukov@gmail.com>"
+    ],
+    "devDependencies": {
+        "libphonenumber-js": "^1.10.0"
```

Comment:
Let's fixate this `devDependency` (**not `peerDependencies`!**) for renovate bot ?